### DISCUSSION TITLE
请教armbian的iptables/nftables设置问题

### DISCUSSION THREAD

**Discussion by LSP0123** [ophub/amlogic-s9xxx-armbian#2056] [Other] [answered]
我正在使用zerotier连接两个子网（分别在不同地区），但是armbian的iptables/nftable设置保存后找不到保存的位置
在ubuntu22.04中，iptables-save会保存设置到/etc/iptables/rules.v4
但是在armbian中没有这个位置。find查找没有查找到对应文件

如果有知道如何设置的大佬，麻烦您指导下，非常感谢

**Comment by upsangel** [comment, marked as answer]
可以安裝和使用 iptables-persistent 來實現
第一次安裝 系統會問你，是否包把現在的 iptables 儲存，如果選是，就會儲存爲 /etc/iptables/rules.v4 和 v6

**Reply by LSP0123** [reply]
感谢，已经解决了
